getting-started-with-directus-and-flask/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: 'Getting Started With Directus And Flask'
+description: 'This guide will show you how to create an application using Flask framework and Directus as a headless CMS'
+author:
+  name: 'Guilherme Souza'
+  avatar_file_name: 'guilherme-souza-avatar.png'
+---
+
+## Introduction
+
+[Flask](https://flask.palletsprojects.com/en/3.0.x/) is a well-known Python framework used to build web APIs, and Directus is a powerful tool that helps developers and their teams integrate data with new or existing applications. In this tutorial, you will learn how to create a simple website using Directus as a [headless CMS](https://directus.io/solutions/headless-cms), creating and retrieving data such as global metadata, dynamic pages, and blog posts.
+
+## Before You Start
+
+You will need:
+
+- To have Python installed on your machine
+- A Directus project - [follow our quickstart guide](https://docs.directus.io/getting-started/quickstart) if you don't already have one.
+- Basic knowledge of Python and Flask
+
+## Setting Up A Flask Project
+
+To create a new Flask project using `venv`, create your project directory and enter it, then run the following commands:
+
+```sh
+python3 -m venv .venv
+source .venv/bin/activate  # On Windows you should use `.venv\Scripts\activate`
+pip install Flask requests python-dotenv
+```
+
+To make *.env* file variables available on the project, create a file called *config.py* and add the following content to it:
+
+```py
+from dotenv import load_dotenv
+
+load_dotenv()
+```
+
+Then create an *app.py* file to start the Flask app:
+
+```py
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.get("/")
+def home():
+    return "Hello world"
+```
+
+And run the following command to run your flask server, which will start a server at [http://localhost:3000](http://localhost:3000):
```

Comment:
```suggestion
And run the following command to run your flask server, which will start a server at `http://localhost:3000`:
```

This is one just because our website rendering throws errors with 404s. Otherwise I'd be quite happy with the link.